job_creator/jobcreator/utils.py

```diff
@@ -59,7 +59,38 @@ def load_kubernetes_config() -> None:
             config.load_kube_config()
 
 
-def ensure_ceph_path_exists(ceph_path: Path) -> Path:
+def create_ceph_mount_path_simple(
+    user_number: str | None = None,
+    experiment_number: str | None = None,
+    mount_path: str = "/isis/instrument",
+    local_ceph_path: str = "/ceph",
+) -> Path:
+    """
+    Creates the ceph mount for the job to output to
+    :param user_number: str, The user number that owns the job
+    :param experiment_number: str, The experiment number that owns the job
+    :param mount_path: str, the path that should be pointed to by default, before RBNumber, and Instrument specific
+    directories.
+    :param local_ceph_path: str, the path that we expect Ceph to be present locally, by default it's /ceph, mostly for
+    testing.
+    :return: str, the path that was created for the mount
+    """
+    initial_path = Path(local_ceph_path) / "GENERIC" / "autoreduce"
```

Comment:
As a side note, I think plotting-service and data-viewer might need an update to handle these new paths

Reply:
Almost definetely.